sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 ### Features Added
 
+- Unhide the `IsIPv6Enabled` parameter for server creation and updating.
```

Comment:
You should not update the already released entry.
Please move the content to the latest entry and fill in the expecting release date.